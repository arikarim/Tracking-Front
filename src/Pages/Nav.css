.nav {
  background: #8bbedb;
}

.toglle {
  border: none;
}

.logout-btn {
  cursor: pointer;
  background: none;
  border: none;
  color: white;
}
