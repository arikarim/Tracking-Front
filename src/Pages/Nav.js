import React from "react";
import { Link, useHistory } from "react-router-dom";
import { logout } from "../PureFunctions/Logout";
import "./Nav.css";
const Nav = ({ setUser }) => {
  const history = useHistory();
  const toke = JSON.parse(localStorage.getItem("token"));
  return (
    <nav className="">
      {toke === "" ? (
        <h3 className="py-3 text-center text-white">Track.it</h3>
      ) : (
        // <ul className="list-unstyled d-flex py-3 justify-content-end">
        //   <li className="mx-3">
        //     <Link className="text-decoration-none link-light" to="/signup">
        //       Sign Up
        //     </Link>
        //   </li>
        //   <li className="mx-3">
        //     <Link className="text-decoration-none link-light" to="/login">
        //       Login
        //     </Link>
        //   </li>
        // </ul>
        <h3 className="py-3 text-center text-white">Track.it</h3>
        // <ul className="list-unstyled d-flex py-3 justify-content-end">
        //   <li className="mx-3">
        //     <Link className="text-decoration-none link-light" to="/">
        //       Home
        //     </Link>
        //   </li>
        //   <li className="mx-3">
        //     <Link className="text-decoration-none link-light" to="/addrecord">
        //       Add Record
        //     </Link>
        //   </li>
        //   <li className="mx-3">
        //     <button
        //       className="logout-btn"
        //       type="submit"
        //       onClick={(e) => logout(setUser, history)}
        //     >
        //       Log out
        //     </button>
        //   </li>
        // </ul>
      )}
    </nav>
  );
};

export default Nav;
