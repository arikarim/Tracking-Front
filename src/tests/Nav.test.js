import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Nav from '../Pages/Navv';
import createTestStore from '../Reducers/index';

describe('Snapshots', () => {
  let store;
  beforeEach(() => {
    store = createTestStore;
  });

  test('should match with snapshot', () => {
    const jsx = (
      <Provider store={store}>
        <BrowserRouter>
          <Nav />
        </BrowserRouter>
      </Provider>
    );
    const tree = (() => renderer.create(jsx).toJSON());
    expect(tree).toMatchSnapshot();
  });
});
