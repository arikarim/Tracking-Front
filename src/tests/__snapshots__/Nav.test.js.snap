// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots should match with snapshot 1`] = `[Function]`;
